include/CppUTest/StandardCLibrary.h

```diff
@@ -1,8 +1,10 @@
 
+/* Must include this first to ensure the StandardC include in CppUTestConfig still happens at the right moment */
+#include "CppUTestConfig.h"
```

Comment:
Perhaps StandardCLibrary.h should `#error` if not included from (or at least after) CppUTestConfig.h?  Then the test could be changed to include TestHarness.h as @arstrube suggests and we'd still have a clear indication to tell people not to include StandardCLibrary.h directly.